data/tlg4090/tlg002/__cts__.xml

```diff
@@ -1,8 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4090" xml:lang="grc" urn="urn:cts:greekLit:tlg4090.tlg002">
-  <ti:title xml:lang="lat">Sancti patris nostri Cyrilli archiepiscopi Alexandrini in D. Joannis evangelium, vol. 1</ti:title>
+  <ti:title xml:lang="lat">Sancti patris nostri Cyrilli archiepiscopi Alexandrini in D. Joannis evangelium</ti:title>
   
 <ti:edition urn="urn:cts:greekLit:tlg4090.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg4090.tlg002" xml:lang="grc">
-    <ti:label xml:lang="lat">Sancti patris nostri Cyrilli archiepiscopi Alexandrini in D. Joannis evangelium</ti:label>
-    <ti:description xml:lang="mul">Cyril, Saint Patriarch of Alexandria, Sancti patris nostri Cyrilli archiepiscopi Alexandrini in D. Joannis evangelium, vol. 1, Pusey, Harvard College Library, 2017</ti:description>
-  </ti:edition>
+    <ti:label xml:lang="mul">εἰς τὸ κατὰ Ἰωάννην εὐαγγέλιον (Vol. 1)</ti:label>
```

Comment:
Perhaps rather than Vol 1, list what books of the work this is?

Reply:
Good idea. I was not sure if we should even push this as it is incomplete? but it is also quite large so I hate to see it skipped.